mesonbuild/modules/pkgconfig.py

```diff
@@ -395,13 +402,19 @@ def generate(self, state, args, kwargs):
         deps.add_priv_reqs(kwargs.get('requires_private', []))
         deps.add_cflags(kwargs.get('extra_cflags', []))
 
+        for dir in subdirs:
```

Comment:
__[Flake8]__ [A001] "dir" is a python builtin and is being shadowed, consider renaming the variable ([link](https://sider.review/gh/repos/19784232/pulls/5775#issue-4419191))

<sub>posted by Sider</sub>